docs/application/native/guides/multimedia/media-conversions.md

```diff
@@ -1,17 +1,9 @@
 # Media Conversions
 
 
-You can perform various media conversions through codecs. You can encode and decode video and audio data, and, in mobile applications, you can also use video transcoding features, such as multi-format codec and output format.
+You can perform various media conversions through codecs. You can encode and decode video and audio data.
 
-The main features of the Media Codec and Video Util API include:
-
-- Media codecs
-
-  You can prepare the [media codecs](#codecs), fill the media packet with data, and run the media codec.
-
-- Video transcoding **in mobile applications only**
-
-  You can prepare the video utilities, manage and monitor [video transcoding](#video), and set parameters.
+You can prepare the [media codecs](#codecs), fill the media packet with data, and run the media codec.
```

Comment:
Suggest combining both the sentences:

You can perform various media conversions through codecs. You can encode and decode video and audio data. To do so, you can prepare the [media codecs](#codecs), fill the media packet with data, and run the media codec.